src/containers/lists/BidList/BidList.tsx

```diff
@@ -47,83 +62,112 @@ const BidList: React.FC<BidListProps> = ({
 		}
 	}, [library]);
 
-	// useEffect(() => {
-	// 	console.log({ blockNumber });
-	// }, [blockNumber]);
-
 	const sorted = bids
 		.slice()
-		.sort((a: Bid, b: Bid) => b.date.getTime() - a.date.getTime());
+		.sort((a: Bid, b: Bid) => Number(b.timestamp) - Number(a.timestamp));
```

Comment:
Could be handy to add this as a function in `lib/utils/bids.ts` - no biggie though